doc/reconstruction_models_list.rst

```diff
@@ -67,137 +67,139 @@
+     - At least one b=0, minimum of 39 acquisitions with spherical and linear encoding; optimal 120 (see :cite:t:`Morez2023`), ideal 217 see Table 1 in :cite:t:`Herberthson2021`
+     - :cite:t:`Herberthson2021`, :cite:t:`Morez2023`
    * - Ball & Stick
      - :bdg-success:`Yes`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
-     - Three b=0, 60 evenly distributed directions per `Jones 1999 <https://doi.org/10.1002/(SICI)1522-2594(199909)42:3%3C515::AID-MRM14%3E3.0.CO;2-Q>`__ at b-value 1000mm/s^2
-     - `Behrens 2003 <https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.10609>`__
+     - Three b=0, 60 evenly distributed directions per :cite:t:`Jones1999` at b-value 1000 s/mm^2
+     - :cite:t:`Behrens2003`
    * - :ref:`QTau-MRI <sphx_glr_examples_built_reconstruction_reconst_qtdmri.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      - Minimum 200 volumes of multi-spherical dMRI (multi-shell, multi-diffusion time; varying gradient directions, gradient strengths, and diffusion times)
-     - Fick 2017
+     - :cite:t:`Fick2018`
    * - Power Map
      - :bdg-success:`Yes`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      - HARDI data with 60 directions at b-value = 3000 s/mm^2, 7 b=0 (Minimum: HARDI data with at least 30 directions)
-     - `DellAcqua2014 <http://archive.ismrm.org/2014/0730.html>`__
+     - :cite:t:`DellAcqua2014`
    * - :ref:`SMT / SMT2 <sphx_glr_examples_built_reconstruction_reconst_msdki.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
-     - 72 directions at each of 5 evenly spaced b-values from 0.5 to 2.5 ms/μm2, 5 b-values from 3 to 5 ms/μm2, 5 b-values from 5.5 to 7.5 ms/μm2, and 3 b-values from 8 to 9 ms/μm2 /  b=0 ms/μm^-2, and along 33 directions at b-values from 0.2–3 ms/μm^-2 in steps of 0.2 ms/μm^−2 (24 point spherical design and 9 directions identified for rapid kurtosis estimation)
-     - `NetoHe2019 <https://doi.org/10.1002/mrm.27606>`__, `Kaden2016b <https://www.nature.com/articles/sdata201672>`__
+     - 72 directions at each of 5 evenly spaced b-values from 0.5 to 2.5 ms/μm^2, 5 b-values from 3 to 5 ms/μm^2, 5 b-values from 5.5 to 7.5 ms/μm^2, and 3 b-values from 8 to 9 ms/μm^2 / b=0 ms/μm^2, and along 33 directions at b-values from 0.2–3 ms/μm^2 in steps of 0.2 ms/μm^2 (24 point spherical design and 9 directions identified for rapid kurtosis estimation)
+     - :cite:t:`Kaden2016b`, :cite:t:`NetoHenriques2019`
    * - :ref:`CTI <sphx_glr_examples_built_reconstruction_reconst_cti.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      -
-     - `NetoHe2020 <https://www.sciencedirect.com/science/article/pii/S1053811920300926>`__, `NovelloL2022 <https://pubmed.ncbi.nlm.nih.gov/35339682/>`__, `NetHe2021 <https://onlinelibrary.wiley.com/doi/10.1002/mrm.28938>`__
+     - :cite:t:`NetoHenriques2020`, :cite:t:`NetoHenriques2021b`, :cite:t:`Novello2022`
 
+
+.. footbibliography::
```

Comment:
not sure we want that. I need to render and check, it seems wrong to me

Reply:
I had concerns as well, but renders correctly locally for me:

<img src="https://github.com/user-attachments/assets/60ccd1ae-b352-45e5-986f-4c6d10bfc72a" width="500">

Have not tried with `.. bibliography::` (IIRC we can only have one call to this and we already have one). Let me know the result once you try that.

Reply:
I confirm, it renders correctly in both case (with and without  `..bibliography::`)

So I will recommend to remove it since it does not seems to be needed